### ISSUE TITLE
GTK System Tray UI

### ISSUE THREAD

**Issue by Xylemon** [screenfreeze/deskcon-desktop#15]
This would be really nice for those who don't use Gnome 3 or Unity like myself. A GTK2 option would be preferable but not required.

**Comment by marble-sh** [comment]
As someone who doesn't use GTK either, I would like to see this as well.

**Comment by mxmehl** [comment]
I agree to the user above: Please add a GTK functionality for people not using Gnone Shell/Unity.
Thanks!
